department-of-veterans-affairs/va.gov-cms
# VACMS-3149: Remove field from topics vocab.

## Description
See #3149 

## Testing done
Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/96617433-9dcf3780-12d1-11eb-944f-c81e383f16b9.png)


## QA steps
As an admin, go to `/admin/structure/taxonomy/manage/topics/overview/fields` and visually verify there are no fields.
